### ISSUE TITLE
I cannot execute

### ISSUE THREAD

**Issue by galigaligo** [loner02/vrm-player#3]
DEMO cannot execute,
Clone CODE ,
After running with NodeJS
console.log("hello");
No content when accessing 3000 port

**Comment by loner02** [comment]
Sorry, code is for demo on glitch.com.
node_modules folder not included on github repo.
You must first install dependencies using 'npm install'.
Then modify package.json. 'start' should be './bin/www' if run locally.
Then run with 'npm start'

**Comment by loner02** [comment]
In addition, assets such as vrm model and animations are loaded from glitch.com

You may however modify the javascript code to load local assets.

**Comment by galigaligo** [comment]
Thank you very much for sharing the code with your reply!

After a few modifications to the code, it's ready to run!

But what I found was,

Using FBX in your code works fine, but the FBX I downloaded from Mixamo doesn't.

Here's your Bellydancing

![Bellydancing-fbx](https://user-images.githubusercontent.com/89366654/151651078-258a9655-77be-4df9-8693-e6ce8ebdd102.png)

This is RumbaDancing I downloaded
![RumbaDancing-fbx](https://user-images.githubusercontent.com/89366654/151651089-ab555990-9087-4072-a884-ecd2d768779b.png)

Why is that?

FBX-link:https://github.com/galigaligo/mytest

**Comment by loner02** [comment]
I'll test this on my end when i get the chance.
But the belly dancing FBX I used was also downloaded from Mixamo.

**Comment by loner02** [comment]
Somehow, it worked for me.

I downloaded Rumba Dancing from Mixamo. Didn't change any option, used FBX binary.

Then added this in "!motions.json":
		{
			"name": "Rumbadance",
			"url": "/assets/motions/Rumba Dancing.fbx",
			"format": "FBX"
		},

Rumba dancing is OK.

![image](https://user-images.githubusercontent.com/11499775/151684399-326ddb6e-e5b4-47af-8fe2-dc7b9760ed42.png)

I downloaded your FBX from mytest, and it is screwed up as you said. It seems like the rotation axes/orientation are not the same. Maybe not all Mixamo animations are created equal? ;)
